ci/jobs/reboot-worker-management-dev.yml.j2

```diff
@@ -0,0 +1,18 @@
+jobs:
+- name: reboot-management-dev-worker-nodes
+  plan:
+  - get: aws-concourse
+    trigger: false
+  - .: (( inject meta.plan.create-aws-profiles ))
+    config:
+      params:
+        AWS_ACC: "{{accounts['management-dev']}}"
+  - .: (( inject meta.plan.fetch-credentials ))
+    config:
+      params:
+        USERNAME_KEY: "/ci/2f0968b1-63a9-a026-102c-36550c89e0ca-user"
+        PASSWORD_KEY: "/ci/2f0968b1-63a9-a026-102c-36550c89e0ca-password"
```

Comment:
We should never hardcode parameters in open-source repositories, not even as a temporary measure.